.yamato/_abv.yml

```diff
@@ -0,0 +1,612 @@
+    dependencies:
+      -  path: .yamato/_editor.yml#editor:priming:CUSTOM-REVISION:windows
+         rerun: on-new-revision
+    artifacts:
+        logs:
+            paths:
+              -  "**/test-results/**"
+smoke_test_editmode_CUSTOM-REVISION:
+    name: SRP Smoke Test - editmode_CUSTOM-REVISION
+    agent:
+        type: Unity::VM
+        image: sdet/gamecode_win10:stable
+        flavor: b1.large
+    variables:
+        UPM_REGISTRY: https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates
+        CUSTOM_REVISION: custom_revision_not_set
+    commands:
+      -  git clone git@github.cds.internal.unity3d.com:unity/utr.git TestProjects/SRP_SmokeTest/utr
+      -  pip install unity-downloader-cli --extra-index-url https://artifactory.internal.unity3d.com/api/pypi/common-python/simple --upgrade
+      -  cd TestProjects/SRP_SmokeTest && unity-downloader-cli --source-file ../../unity_revision.txt -c editor --wait --published-only
+      -  cd TestProjects/SRP_SmokeTest && utr/utr --suite=editor --platform=editmode --testproject=. --editor-location=.Editor --artifacts_path=test-results
+    dependencies:
+      -  path: .yamato/_editor.yml#editor:priming:CUSTOM-REVISION:windows
+         rerun: on-new-revision
+    artifacts:
+        logs:
+            paths:
+              -  "**/test-results/**"
+all_smoke_tests_CUSTOM-REVISION:
+    name: All Smoke Tests - CUSTOM-REVISION
+    dependencies:
+      -  path: .yamato/_abv.yml#smoke_test_Standalone_CUSTOM-REVISION
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_playmode_CUSTOM-REVISION
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_editmode_CUSTOM-REVISION
+         rerun: on-new-revision
+    variables:
+        CUSTOM_REVISION: custom_revision_not_set
+all_project_ci_CUSTOM-REVISION:
+    name: _ABV for SRP repository - CUSTOM-REVISION
+    dependencies:
+      -  path: .yamato/_packages.yml#all_package_ci_CUSTOM-REVISION
+         rerun: always
+      -  path: .yamato/all-universal.yml#All_Universal_CUSTOM-REVISION
+         rerun: always
+      -  path: .yamato/all-universal_stereo.yml#All_Universal_Stereo_CUSTOM-REVISION
+         rerun: always
+      -  path: .yamato/all-shadergraph.yml#All_ShaderGraph_CUSTOM-REVISION
+         rerun: always
+      -  path: .yamato/all-shadergraph_stereo.yml#All_ShaderGraph_Stereo_CUSTOM-REVISION
+         rerun: always
+      -  path: .yamato/all-hdrp.yml#All_HDRP_CUSTOM-REVISION
+         rerun: always
+      -  path: .yamato/all-hdrp_standalone.yml#All_HDRP_Standalone_CUSTOM-REVISION
+         rerun: always
+      -  path: .yamato/all-vfxmain.yml#All_VFXMain_CUSTOM-REVISION
+         rerun: always
+      -  path: .yamato/all-vfx_lwrp.yml#All_VFX_LWRP_CUSTOM-REVISION
+         rerun: always
+    variables:
+        CUSTOM_REVISION: custom_revision_not_set
+trunk_verification_CUSTOM-REVISION:
```

Comment:
for upgrade reasons, can we keep a copy of the original abv script in the repo after you merge this, even if it only contains this job? i can explain on slack :)